website/src/pages/skip-nav.mdx

```diff
@@ -64,15 +64,30 @@ Renders a link that remains hidden until focused to skip to the main content.
 
 #### SkipNavLink Props
 
-| Prop                                   | Type   | Required |
-| -------------------------------------- | ------ | -------- |
-| [`a` props](#skipnavlink-anchor-props) | Cool   |          |
-| [`children`](#skipnavlink-children)    | `node` | false    |
+| Prop                                   | Type    | Required |
+| -------------------------------------- | ------- | -------- |
+| [`a` props](#skipnavlink-anchor-props) | Cool    |          |
+| [`contentId`](#skipnavlink-contentid)  | `string`| false    |
```

Comment:
Maybe also worth to add documentation about the default value for the contentId, which is `reach-skip-nav`

Reply:
Good call! I added a column to the table 'Default'👌🏻 Lmk if that format is alright